apps/connector/tailwind.config.ts

```diff
@@ -0,0 +1,291 @@
+import * as colors from '@status-im/colors'
+import { scrollbarWidth } from 'tailwind-scrollbar-utilities'
+import { fontFamily } from 'tailwindcss/defaultTheme'
+import plugin from 'tailwindcss/plugin'
+
+import type { Config } from 'tailwindcss'
```

Comment:
Should we use the existing config in `tailwind.config.ts` file like we did for the `status-website` repo?